src/transaction/log_recovery.c

```diff
@@ -1049,6 +1049,74 @@ log_recovery_finish_transactions (THREAD_ENTRY * const thread_p)
   // final recovery phase flag is set in the caller
 }
 
+/* log_recovery_analysis_from_transaction_table_snapshot - perform recovery for a passive transaction server
+ *                  starting from a [recent] transaction table snapshot relayed via log from the active
+ *                  transaction server
+ *
+ * most_recent_transaction_table_snapshot_lsa (in): the lsa where a record containing, as payload, the packed contents
+ *                                of a recent transaction table snapshot; starting from that snapshot, analyze the log
+ *                                to construct an actual starting transaction table
+ */
+void
+log_recovery_analysis_from_transaction_table_snapshot (THREAD_ENTRY * thread_p,
+						       log_lsa most_recent_transaction_table_snapshot_lsa)
```

Comment:
A shorter name? Maybe `last_trantable_snapshot_lsa`?
```suggestion
						       log_lsa most_recent_transaction_table_snapshot_lsa)
```

Reply:
shortened it a bit, but I think `most_recent` instead of `last` conveys the meaning better.